packages/kit/src/exports/vite/preview/index.js

```diff
@@ -128,9 +128,14 @@ export async function preview(vite, vite_config, svelte_config) {
 
 				const { pathname, search } = new URL(/** @type {string} */ (req.url), 'http://dummy');
 
-				let filename = normalizePath(
-					join(svelte_config.kit.outDir, 'output/prerendered/pages' + pathname)
-				);
+				let filename = normalizePath(join(svelte_config.kit.outDir, 'output/prerendered/pages' + pathname));
```

Comment:
the formatting seems to have changed here for some reason :thinking:

Reply:
good catch! Thank you for pointing that one out